style.css

```diff
@@ -225,13 +225,122 @@ button:active {
   gap: 10px;
 }
 
+.popupSkillsMobile li {
+  font-size: 12px;
+  font-weight: 600;
+  line-height: 16px;
+  letter-spacing: 0.36px;
+  background-color: #ebecf0;
+  color: #344563;
+  padding: 10px 10px;
+  border-radius: 5px;
+  margin-right: 1%;
+}
+
 .card_content ul li {
   background-color: #eee;
   padding: 8px 12px;
   font-size: 12px;
   color: #3a4a42;
 }
 
+.popupSection {
+  position: fixed;
+  top: 1%;
+  bottom: 0;
+  display: none;
+  width: 100%;
+  height: 100vh;
+  overflow: auto;
+}
+
+.popup {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fff;
+  justify-content: center;
+  padding: 75px 0;
+  border-radius: 15px;
+  width: 80%;
+  margin-left: 10%;
+}
+
+.popup .snapshootMobile {
+  text-align: center;
+  width: 95%;
+}
+
+.popup .snapshootDesktop {
+  display: none;
+}
+
+.popupContent {
+  width: 80%;
+  margin-top: 25px;
+}
+
+.titleAndButtons h2 {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+}
+
+.popupSkills ul {
+  list-style: none;
+}
+
+.popupSkillsDesktop {
+  display: none;
+}
+
+.popupSkillsMobile {
+  display: flex;
+  margin: 15px 0;
+}
+
+.popupDescriptionDesktop {
+  display: none;
+}
+
+.popupDescriptionMobile {
+  font-size: 16px;
+  font-weight: 400;
+  line-height: 24px;
+}
+
+.popupDescriptionMobile p {
+  margin: 15px 0;
+}
+
+.popupButtons {
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] Keep doing the great work 👍 
![Capture](https://github.com/GhausAliKhan/My-Portfolio/assets/80458199/6e45f393-ab8a-43d8-8c41-f188539e728f)